pkg/build/util/util.go

```diff
@@ -74,3 +75,9 @@ func NameFromImageStream(namespace string, ref *kapi.ObjectReference, tag string
 func IsBuildComplete(build *buildapi.Build) bool {
 	return build.Status.Phase != buildapi.BuildPhaseRunning && build.Status.Phase != buildapi.BuildPhasePending && build.Status.Phase != buildapi.BuildPhaseNew
 }
+
+// ConfigSelector returns a label Selector which can be used to find all
+// builds for a BuildConfig.
+func ConfigSelector(name string) labels.Selector {
+	return labels.Set{buildapi.BuildConfigLabel: name}.AsSelector()
```

Comment:
@deads2k why are we using a different label here: https://github.com/openshift/origin/blob/663ca6f814b1130a9eadd5de7a0a2fa420e4eeaa/pkg/cmd/util/clientcmd/factory.go#L204